README.md

```diff
@@ -1,5 +1,20 @@
 # Astro Image Auto-Caption Component
 
+
+If you would like to cite this project, please use the following BibTeX entry:
+
+```bibtex
+@misc{astro_image_autocaption,
+  author = {Kavish Soam},
```

Comment:
@kavishsoam I will add you as one of the authors to the BibTex citation if you do more contributions to the repo other than minor documentation changes. For right now, I'll add an acknowledgments section in the README with a link to your GitHub profile to acknowledge your contributions.
```suggestion
  author = {Nicolas Perez},
```